src/Components/Publishing/EditorialFeature/Components/Vanguard2019/Components/ArtistWrapper.tsx

```diff
@@ -28,37 +28,49 @@ export class VanguardArtistWrapper extends React.Component<
     const { article } = this.props
     const { hero_section, layout, slug, title } = article
     const { isExpanded } = this.state
+    const textColor = isExpanded ? "white" : undefined
 
     return (
-      <ArtistContainer pb={4} maxWidth={1000} px={4} mx="auto">
-        <Box textAlign="center">
-          <ArtistTitle size="8">{title}</ArtistTitle>
-          <Box position="absolute">
-            <Share
-              // TODO: We may need to use custom urls for in-page routing
-              url={getFullEditorialHref(layout, slug)}
-              title={title}
-            />
+      <Box background={isExpanded && "black"}>
+        <ArtistContainer pb={4} maxWidth={1000} px={4} mx="auto">
+          <Box textAlign="center">
+            <ArtistTitle size="8" color={textColor}>
+              {title}
+            </ArtistTitle>
+            <Box position="absolute">
+              <Share
+                // TODO: We may need to use custom urls for in-page routing
+                url={getFullEditorialHref(layout, slug)}
+                title={title}
+                color={textColor}
+              />
+            </Box>
+            {hero_section && (
+              <Sans size="4" weight="medium" color={textColor}>
+                {hero_section.deck}
+              </Sans>
+            )}
           </Box>
-          {hero_section && (
-            <Sans size="4" weight="medium">
-              {hero_section.deck}
-            </Sans>
-          )}
-        </Box>
-        {/** TODO: Sections may need to be customized to handle expansion */}
-        <Sections article={article} customWidth={900} />
 
-        <ReadMoreButton
-          size="5"
-          weight="medium"
-          textAlign="center"
-          onClick={this.onExpand}
-        >
-          {isExpanded ? "Read More" : "Close"}
-          <Sans size="8">{isExpanded ? "\u2193" : "\u2191"}</Sans>
-        </ReadMoreButton>
-      </ArtistContainer>
+          <Sections
+            article={article}
+            customWidth={900}
+            isTruncatedAt={!isExpanded && 2}
+            color={textColor}
+          />
+
+          <ReadMoreButton
+            size="5"
+            weight="medium"
+            textAlign="center"
+            onClick={this.onExpand.bind(this)}
+            color={textColor}
+          >
+            {isExpanded ? "Close" : "Read More"}
+            <Sans size="8">{isExpanded ? "\u2191" : "\u2193"}</Sans>
```

Comment:
I wonder if we should use the icons in Palette instead of unicode.  I know the icon is slight different than our up/down arrows, but I'm thinking about 1) browser font support or lack thereof and also if we should stay consistent with the icons in our design system.

Reply:
Heads up these had already been merged, I just reversed the order -- i'm open to using icons, going to check in with editorial to see how they feel about using our system icons. Are there known issues with unicode errors? They seem to be fine in most fonts and have been working without issue in the feature so far.

Reply:
I think it will be fine in most browsers, but from what I recall no single browser can decode every Unicode char (without a software install/update of some sort at least) so users running outdated or less popular browsers might see the text  `"\u2191"` instead of what it compiles to. But my suggestion to use Palette's icons was more about consistency throughout Artsy and using our design system moreso than browser issues.